app/models/conversation.rb

```diff
@@ -160,7 +156,8 @@ def activity_message_params(content)
 
   def notify_status_change
     {
-      CONVERSATION_RESOLVED => -> { saved_change_to_status? && resolved? && assignee.present? },
+      CONVERSATION_REOPENED => -> { saved_change_to_status? && open? },
```

Comment:
If the status is a bot then the conversation is not reopened. I think we should check for bot_status. Also, should we specify that the conversation is reopened? I would prefer `opened` over `reopened`